examples/systrack.lua

```diff
@@ -3,41 +3,31 @@
 -- SPDX-License-Identifier: MIT OR GPL-2.0-only
 --
 
-local linux  = require("linux")
-local probe  = require("probe")
-local device = require("device")
-local systab = require("syscall.table")
-
-local syscalls = {"openat", "read", "write", "readv", "writev", "close"}
+local lunatik = require("lunatik")
+local linux   = require("linux")
+local device  = require("device")
+local rcu     = require("rcu")
 
 local function nop() end -- do nothing
 
 local s = linux.stat
 local driver = {name = "systrack", open = nop, release = nop, mode = s.IRUGO}
 
-local track = {}
+local systrack = rcu.table()
+lunatik._ENV.systrack = systrack 
+
 local toggle = true
 function driver:read()
 	local log = ""
 	if toggle then
-		for symbol, counter in pairs(track) do
-			log = log .. string.format("%s: %d\n", symbol, counter)
-		end
+		rcu.map(systrack, function (symbol, counter)
+			log = log .. string.format("%s: %d\n", symbol, counter:getnumber(0))
+		end)
 	end
 	toggle = not toggle
 	return log
 end
 
-for _, symbol in ipairs(syscalls) do
-	local address = systab[symbol]
-	track[symbol] = 0
-
-	local function handler()
-		track[symbol] = track[symbol] + 1
-	end
-
-	probe.new(address, {pre = handler, post = nop})
-end
-
+driver.probes = lunatik.runtime("systrack/probes", false)
```

Comment:
Shouldn’t it be `"examples/systrack/probes"`? As is, I get the message `Cannot open /lib/modules/lua/systrack/probes.lua`.

Reply:
Yeah! Good catch.. thanks! I'll fix it.. I was using Lua root for testing.. sorry about that